packages/test/local-server-stress-tests/src/stressDataObject.ts

```diff
@@ -45,7 +46,20 @@ export interface CreateChannel {
 	tag: `channel-${number}`;
 }
 
-export type StressDataObjectOperations = UploadBlob | CreateDataStore | CreateChannel;
+export interface EnterStagingMode {
+	type: "enterStagingMode";
+}
+export interface ExitStagingMode {
+	type: "exitStagingMode";
+	accept: boolean;
```

Comment:
Nit: minimize terminology used
```suggestion
	commit: boolean;
```